src/appMain/CMakeLists.txt

```diff
@@ -162,7 +162,7 @@ set (HEADERS
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   if(NOT QT_PORT)
-    list(APPEND LIBRARIES sqlite3 bthprops)
+    list(APPEND LIBRARIES sqlite3 bthprops Ws2_32)
```

Comment:
in the files: src\components\transport_manager\src\tcp\tcp_client_listener.cc :36
src\components\transport_manager\src\tcp\tcp_device.cc: 38
src\components\transport_manager\src\tcp\tcp_socket_connection.cc :43
yet included "#pragma comment(lib, "Ws2_32.lib")"